### DISCUSSION TITLE
Open available database with Floor

### DISCUSSION THREAD

**Discussion by bxTruong** [vitusortner/floor#458] [Q&A] [answered]
Help me

**Comment by vitusortner** [comment, marked as answer]
Hi! What are you wanting to do? In case you want to prepopulate your database, #255 might be of help.

**Reply by bxTruong** [reply]
Hello, How to open available database with Floor?

**Reply by mqus** [reply]
could you add a few more details (maybe in pseudocode)? what are the preconditions (what do you mean with "available")? And what do you want to achieve, what is your usecase?

**Comment by neiljaywarner** [comment]
@vitusortner  how about add2app for existing db created by host app, ideally handling encrypted db
